### ISSUE TITLE
0.6.0 seems to be broken in npmjs

### ISSUE THREAD

**Issue by metalim** [tonistiigi/audiosprite#44]
```
└> npm -g i audiosprite
/Users/metalim/.nvm/versions/node/v5.7.0/bin/audiosprite -> /Users/metalim/.nvm/versions/node/v5.7.0/lib/node_modules/audiosprite/cli.js
/Users/metalim/.nvm/versions/node/v5.7.0/lib
└── audiosprite@0.6.0

└> audiosprite
env: node\r: No such file or directory

└> npm -g i audiosprite@0.5
/Users/metalim/.nvm/versions/node/v5.7.0/bin/audiosprite -> /Users/metalim/.nvm/versions/node/v5.7.0/lib/node_modules/audiosprite/cli.js
- balanced-match@0.3.0 node_modules/audiosprite/node_modules/balanced-match
- concat-map@0.0.1 node_modules/audiosprite/node_modules/concat-map
- brace-expansion@1.1.3 node_modules/audiosprite/node_modules/brace-expansion
- inherits@2.0.1 node_modules/audiosprite/node_modules/inherits
- minimatch@3.0.0 node_modules/audiosprite/node_modules/minimatch
- path-is-absolute@1.0.0 node_modules/audiosprite/node_modules/path-is-absolute
- wrappy@1.0.1 node_modules/audiosprite/node_modules/wrappy
- once@1.3.3 node_modules/audiosprite/node_modules/once
- inflight@1.0.4 node_modules/audiosprite/node_modules/inflight
- glob@6.0.4 node_modules/audiosprite/node_modules/glob
/Users/metalim/.nvm/versions/node/v5.7.0/lib
└── audiosprite@0.5.0

└> audiosprite
error: No input files specified.
info: Usage: audiosprite [options] file1.mp3 file2.mp3 *.wav
...
```

**Comment by goldenratio** [comment]
ya.. happens!!

**Comment by zlumer** [comment]
Thank you for the report. Looks like a problem with line endings:
https://github.com/nwjs/nw-builder/issues/75

I'm going to fix this in the next patch. PRs are welcome, of course.

**Comment by CraigBeswetherick** [comment]
I just got hung up on this for quite some time! if i get the chance, will make a PR

**Comment by SargoDarya** [comment]
Fixed through #54
